redirect-rules/simple/README.md

```diff
@@ -0,0 +1,74 @@
+
+```json
+{
+  "has_consented": true
+}
+```
+
+If you don't agree to the consent form, but still submit, you will still end up at http://jwt.io, but the URL will contain the authorization error generated by the rule:
+
+```
+http://jwt.io/#error=unauthorized&error_description=User%20did%20not%20consent!
+```
+
+## Consent Form Setup
+
+If you'd like to play around with your own implementation of the consent form webtask, you can host your own version of the [`webtask.js`](webtask.js) script by following these steps:
+
+1. If you haven't done so already, install the Webtask CLI and configure a container for your Auth0 account by following the steps in the [Webtasks](https://manage.auth0.com/#/account/webtasks) tab of your account settings.
+1. While you're in the same directory as the `webtask.js` script, run the following command to create your webtask instance:  
+  ```bash
+  $ wt create -n consent webtask.js -p "WEBTASK_PROFILE"
+  ```
+
+  where:
+  * `WEBTASK_PROFILE` is the name of the profile you set up in the previous step
+
+  The output of this command will contain the URL of your new webtask.
+
+1. In your account's [Rules](https://manage.auth0.com/#/rules) screen, delete the existing `CONSENT_FORM_URL` setting and recreate it with the value that is the URL that was output in the previous step.
+
+1. Try the rule along with your instance of the webtask by the following the steps in the [Run the Rule](#rule-the-rule) section.
+
+1. If you want make changes to the webtask, you can upload a new version simply by running the same `wt create` command you did before.
+
+## Data Integrity
+
+As stated, this is a very basic example of using a redirect rule to invoke a consent form. The `confirm` field (which has the value of `yes`) that is being passed back to the Auth0 redirect rule is in plain text.
+
+There are scenarios where you need better assurances of the integrity of the data being returned by the external website (in this case the webtask). For example, if you want to be sure that the data truly came from a trusted source, then it should be signed. If the data is sensitive, then it should be encrypted. A good mechanism for doing this is to use a [JWT](http://jwt.io/). You can build a JWT with claims (that you can optionally encrypt) and then sign it with either a secret shared with your Auth0 rule or with a private key, whose public key is known by the rule. The rule can then verify that the claims are legit and decrypt them, if necessary.
```

Comment:
This makes it sound like https://jwt.io is some kind of service that allows you to protect data.